.travis.yml

```diff
@@ -1,4 +1,10 @@
 language: ruby
-cache: bundler
+script:
+  - bundle exec rake rubocop
 rvm:
   - 2.3.3
+sudo: false
+cache: bundler
+script:
```

Comment:
You can't combine script tags like this. This ends up with:

```ruby
YAML.load_file('.travis.yml')
=> {"language"=>"ruby", "script"=>["bundle exec rake", "bin/check-fail.sh"], "rvm"=>["2.3.3"], "sudo"=>false, "cache"=>"bundler"}
```

i.e. losing the `rubocop` check.